TPMCmd/tpm/include/Ossl/TpmToOsslHash.h

```diff
@@ -160,7 +160,7 @@ typedef const BYTE* PCBYTE;
 // Function aliases. The code in CryptHash.c uses the internal designation for the
 // functions. These need to be translated to the function names of the library.
 #  define tpmHashStart_SHA1            \
-    SHA1_Init  // external name of the \
+    SHA1_Init  // external name of the
```

Comment:
clang-format doesn't handle comments on split #defines very well, please move the comment.
This is something I missed when I did the previous format on this branch.

```
// external name of the initialization method
#  define tpmHashData_SHA1           SHA1_Init
```